flang-rt/include/flang-rt/runtime/descriptor.h

```diff
@@ -365,7 +379,18 @@ class Descriptor {
 
   RT_API_ATTRS std::size_t SizeInBytes() const;
 
-  RT_API_ATTRS std::size_t Elements() const;
+  RT_API_ATTRS std::size_t Elements() const {
```

Comment:
Should we mark such methods with force-inline attribute.  I am not suggesting doing it in this PR, but I wonder if you tried it.

Reply:
I didn't try it, and I've run into other trouble because something somewhere has a link-time unresolved external reference now to Descriptor::Elements().  Updating soon to leave Elements out-of-line with a new inline InlineElements() that it can call.

Reply:
I have no clue how this change could have caused a linking issue..